webhooks/clusterclass.go

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
Not entirely sure about the package but I think it's consistent with `controllers/`